server/src/Api/Handlers.hs

```diff
@@ -239,14 +271,61 @@ queryUtxos txInputs =
             $ C.QueryUTxO (C.QueryUTxOByTxIn txInputs)
         )
 
+--------------------------------------------------------------------------------
+-- Set script integrity hash
+--------------------------------------------------------------------------------
+
+setScriptIntegrityHash ::
+  Shelley.ProtocolParameters ->
+  Shelley.TxBody C.AlonzoEra ->
+  Shelley.TxBody C.AlonzoEra
+setScriptIntegrityHash pparams txBodyAlonzo =
+  case txBodyAlonzo of
+    Shelley.ShelleyTxBody _e txBody scripts scriptData _a _v ->
```

Comment:
I think the `_` prefixes might be a bit misleading, since that's conventionally used to indicate a discarded value, but they're actually used below